integration_test/test_xgboost_experiment.py

```diff
@@ -30,16 +27,24 @@
 
 
 class TestXGBoostExperiment:
+  def _generate_randomized_search_space(self):
+    search_space = copy.deepcopy(SEARCH_SPACES)
+    if random.randint(0, 1) == 1:  # add bounds and type to eta randomly
+      search_space[0]['type'] = 'double'
+      search_space[0]['bounds'] = {'min': 0.1, 'max': 0.5}
+    if random.randint(0, 1) == 1:   # add bounds and type to min_child_weight randomly
+      search_space[1]['type'] = 'double'
+      search_space[1]['bounds'] = {'min': 0.0, 'max': 0.3}
+    random_subset_size = random.randint(1, len(search_space))
+    search_space = random.sample(search_space, random_subset_size)
+    return search_space
+
   def _form_random_experiment_config(self, task):
     experiment_params = _form_random_run_params(task)
     is_classification = True if task in ('binary', 'multiclass') else False
-    metric_to_optimize = random.choice(DEFAULT_CLASSIFICATION_METRICS) if is_classification \
-      else random.choice(DEFAULT_REGRESSION_METRICS)
-
-    random_subset_size = random.randint(1, len(SEARCH_SPACES))
-    search_space = random.sample(SEARCH_SPACES, random_subset_size)
-    if not any([p['type'] == 'double' for p in search_space]):
-      search_space.append(SEARCH_SPACES[0])
```

Comment:
We do need to have at least 1 double parameter to avoid triggering pagerduty with this test.

Reply:
Added back, but in  _generate_randomized_search_space function